zipkin-autoconfigure/storage-influxdb-http/src/main/java/zipkin/autoconfigure/storage/influxdb/http/ZipkinInfluxDBHTTPStorageProperties.java

```diff
@@ -0,0 +1,62 @@
+package zipkin.autoconfigure.storage.influxdb.http;
+
+/**
+ * Copyright 2016-2017 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+public class ZipkinInfluxDBHTTPStorageProperties {
+  private String address = "http://localhost:8086";
+  private String password = "";
+  private String username = "root";
+  private String retention_policy = "default";
+  private String database = "zipkin2";
```

Comment:
is this to dodge something? If not, use "zipkin" as when there's no existing storage we don't have to dodge.